include/nil/crypto3/zk/commitments/batched_commitment.hpp

```diff
@@ -173,18 +176,25 @@ namespace nil {
                     }
 
                     void eval_polys() {
-                        for(auto const &[k, poly] : _polys) {
```

Comment:
It's look good as it was.

Reply:
For some reason, I can not capture k and poly by reference in the following lambda, if they are names used in unpacking.